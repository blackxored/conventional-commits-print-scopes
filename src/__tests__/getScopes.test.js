const getScopes = require('../getScopes');

describe('getScopes', () => {
  it('works', () => {
    expect(getScopes()).toMatchSnapshot();
  });
});
