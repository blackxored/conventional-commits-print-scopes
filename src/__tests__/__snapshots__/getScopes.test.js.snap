// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getScopes works 1`] = `
Array [
  "package",
  "release",
]
`;
